docs/reports/2026-05-05-issue-493-static-producer-inventory.md

```
@@ -0,0 +1,146 @@
+# 2026-05-05 issue 493 static producer inventory
+
+## Scope
+
+This report records the first non-legacy static producer inventory for issue
+#493. The inventory is generated from the current decompiled C# source under
```

markdownlint-cli2 (0.22.1)

[warning] No space after hash on atx style heading

(MD018, no-missing-space-atx)